util/Migrator/DbScripts/2022-12-26_00_ProviderAddProviderTypeBillingPhone.sql

```diff
@@ -0,0 +1,178 @@
+-- Add column 'Type' to 'Provider' table
+IF COL_LENGTH('[dbo].[Provider]', 'Type') IS NULL
```

Comment:
I don't believe we want to add `non null` columns like this. Here is a reference to our [SQL code style](https://contributing.bitwarden.com/contributing/code-style/sql) which discusses adding a value using a default constraint instead of multi-step null -> non-null row conversion. @joseph-flinn originally pointed this out to me a while back (thanks!)

Reply:
Thanks for pointing me to the documentation. My first implementation actually had the default constraint but I wasn't sure if that was right and changed it later.
I've added the default constraint.